src/software/simulation/simulator.h

```diff
@@ -110,6 +110,18 @@ class Simulator
     void setYellowRobotPrimitives(ConstPrimitiveVectorPtr primitives);
     void setBlueRobotPrimitives(ConstPrimitiveVectorPtr primitives);
 
+    /**
+     * Sets the primitive being simulated by the robot in simulation
```

Comment:
This has the same description as `setRobotPrimitive`. Might be better to say "Sets the primitive being simulated by a specific team robot in simulation" or something along those lines (just anything to make the description different).

Reply:
Changed to specify "robot on the corresponding team"